lib/tasks/application.rake

```diff
@@ -121,4 +121,31 @@ namespace :application do
     Rails.logger = ActiveSupport::Logger.new($stdout)
     Rails.logger.level = 1
   end
+
+  namespace :links_valid do
+    desc "Checks that all links in markdown for divisions are valid"
+    task divisions: :environment do
+      base_url = "https://theyvoteforyou.org.au"
+
+      Division.all.each do |division|
```

Comment:
Use `find_each` instead of `find` here https://apidock.com/rails/ActiveRecord/Batches/ClassMethods/find_each